src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1) {
```

Comment:
Selectors `&--1 &__star:nth-child(-n + 1)`, `&--2 &__star:nth-child(-n + 2)`, `&--3 &__star:nth-of-type(-n + 3)`, `&--4 &__star:nth-of-type(-n + 4)`, `&--5 &__star:nth-of-type(-n + 5)` are using different pseudo-classes `nth-child` and `nth-of-type`. Stick to one for consistency unless there is a specific reason for the difference.